### PR TITLE
Check for and eliminate duplicate names in Records class

### PR DESCRIPTION
**Pull request by martinholmer**
Add test that checks for duplicate names in Records.NAMES and Records.ZEROED_NAMES.  The new test spotted 12 duplicates in the ZEROED_NAMES list.

### CODE REVIEW

File: taxcalc/tests/test_records.py [hunk 1]
```diff
@@ -81,3 +81,17 @@ def test_imputation_of_cmbtp_itemizer():
                                            e62100, e00700, e04470,
                                            e21040, e18500, e20800)
     assert np.allclose(cmbtp_itemizer, test_itemizer)
+
+
+def test_for_duplicate_names():
+    var_names = set()
+    inp_names = set()
+    for var_name, inp_name in Records.NAMES:
```
Comment (MattHJensen) [inline taxcalc/tests/test_records.py]: @martinholmer, why do you need both `var_names` and `inp_names` for this test?

Comment (MattHJensen) [inline taxcalc/tests/test_records.py]: Sorry, I just figured it out.

**Comment by MattHJensen** [comment]
Looks good. Merging.
